### PR TITLE
Update docs to use symbols in :args, relates to #247

### PR DESCRIPTION
**Pull request by johantonelli**
Updated the docs examples to use symbol instead of keywords in the interest of clearer examples.

### CODE REVIEW

File: docs/queries.adoc [hunk 1]
```diff
@@ -382,7 +382,7 @@ queries to apply range filters using predicates.
          [c :condition/humidity humidity]]
  :order-by [[device-id :asc]]
  :limit 10
- :args [{:my-offset 990}]}
+ :args [{'my-offset 990}]}
```
Comment (refset) [inline docs/queries.adoc:385]: We should quote the outer vecs for `:where` `:find` and `:order-by` as well.

File: docs/queries.adoc [hunk 2]
```diff
@@ -401,7 +401,7 @@ connected to a given entity via the `:follow` attribute.
 ----
 {:find [?e2]
  :where [(follow ?e1 ?e2)]
- :args [{:?e1 :1}]
+ :args [{'?e1 :1}]
```
Comment (refset) [inline docs/queries.adoc]: I don't think we need to quote this particular instance at all.

**Review by refset — CHANGES_REQUESTED** [review]
@Antonelli712 a couple of small changes but otherwise looks good to go :+1:

**Comment by johantonelli** [comment]
Changes made, and updated all quotes for continuity.

**Pull request merged**